pkg/workloads/kubernetes.go

```diff
@@ -104,6 +111,11 @@ func fetchK8sLabels(containerID string, containerLbls map[string]string) (map[st
 // injected by Istio with a sidecar proxy that is compatible with Cilium.
 func isInjectedWithIstioSidecarProxy(containerID string, pod *corev1.Pod) bool {
 	scopedLog := log.WithField(logfields.ContainerID, shortContainerID(containerID))
+
+	if SidecarIstioProxyImageRegexp == nil {
```

Comment:
Can we move this to init code early on? It will be hard to debug if Cilium comes up and then just dies when the first workload is started.

Reply:
Done. I initialized it with the default regexp, so it's now never `nil`.